docs/examples/deadline.rst

```diff
@@ -0,0 +1,68 @@
+.. _examples_deadline:
+
+Deadline
+========
+
+The Deadline QoS raises an alarm when the frequency of new samples falls below a predefined threshold.
+This policy can be useful for those cases which need periodical updates.
+
+It can be distinguished between publisher and subscriber deadline period.
```

Comment:
Who is "it"? Something like there exists a distinction between... would sound better